tests/e2e/expected_results/urllib3/urllib3.json

```diff
@@ -352,10 +352,22 @@
                     ],
                     "justification": [
                         "Successfully verified level 3: ",
-                        "verify passed : urllib3-2.0.3-py3-none-any.whl,verify passed : urllib3-2.0.3.tar.gz"
+                        "verify passed: urllib3-2.0.3-py3-none-any.whl,verify passed: urllib3-2.0.3.tar.gz"
                     ],
                     "result_type": "PASSED"
                 },
+                {
+                    "check_id": "mcn_provenance_witness_level_one_1",
+                    "check_description": "Check whether the target has a level-1 witness provenance.",
+                    "slsa_requirements": [
+                        "Provenance - Available - SLSA Level 1",
+                        "Provenance content - Identifies build instructions - SLSA Level 1",
+                        "Provenance content - Identifies artifacts - SLSA Level 1",
+                        "Provenance content - Identifies builder - SLSA Level 1"
+                    ],
+                    "justification": [],
+                    "result_type": "PASSED"
```

Comment:
Same comment here.

Reply:
Fixed in d234c2736a370d13320b0b7338ba461224f2a31b.